src/add-rmv-edit.js

```diff
@@ -0,0 +1,75 @@
+const todoList = document.getElementById('tasks');
+const form = document.getElementById('form');
+const taskIn = document.getElementById('taskInput');
+
+const updateList = () => {
+  const tasks = JSON.parse(localStorage.getItem('listItems')) || [];
+  if (tasks === null) return;
+  const sortedList = tasks.slice().sort((a, b) => a.index - b.index);
+  todoList.innerHTML = '';
+  sortedList.forEach((task) => {
+    const { completed } = task;
+    const checked = completed ? 'active' : '';
+    const lineT = completed ? 'line' : '';
+    const unfinished = !completed ? 'active' : '';
+    const list = `
+        <div class="task task-${task.index}">
+          <div class="checkbox" data-btn="${task.index}">
+            <button class="square ${unfinished}"></button>
+            <button class="done ${checked}"></button>
+          </div>
+          <input type="text" class="list ${lineT}" value="${task.description}" data-desc="${task.index}"/>
+          <button class="move" data-remove="${task.index}"></button>
+        </div>
+      `;
+    todoList.insertAdjacentHTML('beforeend', list);
+  });
+};
+
+const addTask = (e) => {
+  e.preventDefault();
+  const tasks = JSON.parse(localStorage.getItem('listItems')) || [];
+  const tasksItem = taskIn.value;
+  taskIn.value = '';
+  if (tasksItem === null) return;
+  const task = {
+    description: tasksItem,
+    completed: false,
+    index: tasks.length,
```

Comment:
- [ ] The expected output starts at index 1, not zero. changes this "index: tasks. length," to index: tasks.length+1, and at line 54  task.index = count+1;

![image](https://github.com/binyamolango/toDoListApp-peerReview/assets/61753075/1156fca6-c5f8-4b12-b82c-8014ede793be)